examples/http-proxy/http_proxy.py

```diff
@@ -96,9 +97,16 @@ async def start(self):
             "-c",
             f"echo {shlex.quote(msg)} > /usr/share/nginx/html/index.html",
         )
+        script.run("/bin/rm", "/var/log/nginx/access.log", "/var/log/nginx/error.log")
         script.run("/usr/sbin/nginx"),
         yield script
 
+    async def shutdown(self):
+        script = self._ctx.new_script()
+        script.run("/bin/cat", "/var/log/nginx/access.log")
+        script.run("/bin/cat", "/var/log/nginx/error.log")
+        yield script
```

Comment:
OK, I totally don't get what this is for.
We `cat` something on provider? And ignore results? Why?

(I think this should be explained as a comment in the code, not only to me here)

Reply:
because the results will make it to the logs anyway... but yes, let's explain that in the comments

Reply:
@johny-b added the comment above